management/test_lifecycle.py

```diff
@@ -0,0 +1,104 @@
+import sys
+from unittest import mock
+
+import pytest
+
+# avoid requiring PyGitHub to run tests
+sys.modules["github"] = mock.MagicMock()
+
+
+@pytest.mark.parametrize(
+    "input_tag,want",
+    [
+        ("compute-openff_mw-300-600", ([300.0, 600.0], "compute-openff")),
+        (
+            "compute-pr393_mw-300-600-900",
+            ([300.0, 600.0, 900.0], "compute-pr393"),
+        ),
+        ("compute-pr393", ([], "compute-pr393")),
+    ],
+)
+def test_parse_tags(input_tag, want):
+    from lifecycle import parse_tags
+
+    got = parse_tags(input_tag)
+    assert got == want
+
+
+@pytest.mark.parametrize(
+    "dsname,dstype,bins,want",
+    [
+        (
+            "OpenFF Sulfur Hessian Training Coverage Supplement v1.0",
+            "singlepoint",
+            [100.0, 200.0, 300.0],
+            [6, 220, 234, 439],
```

Comment:
(not blocking) It's kinda confusing to have these be "the number of things in each bin once you sort the bins by population". It would be more straightforward for it to be "the number of things in each bin, where the bins are sorted from lowest MW to largest".

Reply:
Agreed. I updated the code to assert on the mapping of `bin_id` to `len(record_ids)` instead of just these sorted lengths.